lib/mix/tasks/appsignal.install.ex

```diff
@@ -28,6 +28,7 @@ defmodule Mix.Tasks.Appsignal.Install do
         activate_config_for_env("dev")
         activate_config_for_env("stag")
         activate_config_for_env("prod")
+        deactivate_config_for_env("test")
```

Comment:
Right now, the installer will put the `active: true` line in `config/appsignal.exs` if there are no environment based configuration files (dev, prod, stag, or test). If there are, `active` gets set in those, and it'll be omitted from the main config file. 

That way, there's always only one place where the configuration gets set, so it never gets overridden (which is what caused the issue in #208).